supplementary_materials/notebooks/fastdraft-deepseek/fastdraft_deepseek.ipynb

```diff
@@ -0,0 +1,375 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    if not filecmp.cmp(str(model_dir / "openvino_tokenizer.xml"), str(draft_model_path / "openvino_tokenizer.xml"), shallow=False):
</pre><p>probably you can just copy already converted tokenizer from model dir to draft model dir instead of conversion</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2717/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
I though to do that but shutils doesn't work with Windows, what is your preferred way?

Reply:
what kind of problem do you observe for windows?
in some cases you may see permission error on windows if file in use during moving/coping. But if you copy it before starting work with model (before LLMPipeline creation) there should not be an issue

Reply:
Fixed, shutil worked for me now, not sure what the problem was.